growthookServer/src/main/java/com/example/growthookserver/api/actionplan/service/Impl/ActionPlanServiceImpl.java

```diff
@@ -30,14 +32,18 @@ public class ActionPlanServiceImpl implements ActionPlanService {
 
     @Override
     @Transactional
-    public ActionPlanCreateResponseDto createActionPlan(Long seedId, ActionPlanCreateRequestDto actionPlanCreateRequestDto){
+    public void createActionPlan(Long seedId, ActionPlanCreateRequestDto actionPlanCreateRequestDto){
         Seed seed = seedRepository.findSeedByIdOrThrow(seedId);
-        ActionPlan actionPlan = ActionPlan.builder()
-                .content(actionPlanCreateRequestDto.getContent())
-                .seed(seed)
-                .build();
-        ActionPlan savedActionPlan = actionPlanRepository.save(actionPlan);
-        return ActionPlanCreateResponseDto.of(savedActionPlan.getId());
+
+        List<String> contents = actionPlanCreateRequestDto.getContents();
+
+        for(String content : contents) {
+            ActionPlan actionPlan = ActionPlan.builder()
+                    .content(content)
+                    .seed(seed)
+                    .build();
+            actionPlanRepository.save(actionPlan);
+        }
```

Comment:
stream 문법으로 간단하게 나타냈던 것과 통일 시켜서 아래처럼 바꿔도 좋을 것 같아요 ~

```suggestion
        List<String> contents = actionPlanCreateRequestDto.getContents();
        
        contents.stream()
                .map(content -> ActionPlan.builder().content(content).seed(seed).build())
                .forEach(actionPlanRepository::save);

```

Reply:
와아앙 완전 좋아요!!감쟈합니다👍👍